packages/browser/src/request.ts

```diff
@@ -257,15 +274,14 @@ if (navigator?.sendBeacon) {
 }
 
 // This is the entrypoint. It takes care of sanitizing the options and then calls the appropriate request method.
-export const request = (_options: RequestWithOptions) => {
+export const request = (_options: RequestWithOptions): void => {
     // Clone the options so we don't modify the original object
     const options = { ..._options }
     options.timeout = options.timeout || 60000
 
     options.url = extendURLParams(options.url, {
         _: new Date().getTime().toString(),
         ver: Config.LIB_VERSION,
-        compression: options.compression,
```

Comment:
we were always setting compression on the URL  
just now there's a fall back we can't always trust the option and let the transport method decide whether to do it or not